docs/sources/release-notes/v2.3.md

```diff
@@ -0,0 +1,54 @@
+
+- **New Helm chart minor release**: The Mimir Helm chart is the best way to install Mimir on Kubernetes. As part of the Mimir 2.3 release, we’re also releasing version 3.1 of the Mimir Helm chart. Notable enhancements follow. For the full list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+  - We've upgraded the MinIO subchart dependency from a [deprecated chart](https://helm.min.io/) to the [supported one](https://charts.min.io/). This creates a **breaking change** in how the administrator password is set. However, as the built-in MinIO is not a recommended object store for production use cases, this change did not warrant a new major version of the Mimir Helm chart.
+  - The backfill API endpoints for importing historic time series data are now exposed on the Nginx gateway.
+  - Nginx now sets the value of the `X-Scope-OrgID` header equal to the value of Mimir's `no_auth_tenant` parameter by default. The previous release had set the value of `X-Scope-OrgID` to `anonymous` by default which [complicated the process of migrating to Mimir](https://github.com/grafana/mimir/issues/2563).
+  - Memberlist now uses DNS service-discovery by default, which should decrease startup time for large Mimir clusters.
+
+## Upgrade considerations
+
+In Grafana Mimir 2.3 we have removed the following previously deprecated configuration options:
+
+- The `extend_writes` parameter in the distributor YAML configuration and `-distributor.extend-writes` CLI flag have been removed.
+- The `active_series_custom_trackers` parameter has been removed from the YAML configuration. It had already been moved to the runtime configuration. See [#1188](https://github.com/grafana/mimir/pull/1188) for details.
+
+With Grafana Mimir 2.3 we have also updated the default value for `-distributor.ha-tracker.max-clusters` to `100` to provide Denial-of-Service protection. Previously `-distributor.ha-tracker.max-clusters` was unlimited by default which could allow a tenant with HA Dedupe enabled to overload the HA tracker with `__cluster__` label values that could cause the HA Dedupe database to fail.
+
+## Bug fixes
+
+- [PR 2447](https://github.com/grafana/mimir/pull/2447): Fix incorrect mapping of http status codes 429 to 500 when request queue is full in the query-frontend. This corrects behavior in the query-frontend where a 429 "Too Many Outstanding Requests" from a querier would be returned to the user as a 500 system error when the error is retryable.
+- [PR 2505](https://github.com/grafana/mimir/pull/2505): Memberlist KV store now tries to "fast-join" the cluster to avoid serving empty KV store. This fix is to address the confusing "empty ring" error response and the error log message "ring doesn't exist in KV store yet" from services when there are other members in the ring when a service starts.
```

Comment:
```suggestion
- [PR 2505](https://github.com/grafana/mimir/pull/2505): The Memberlist key<>value (KV) store now tries to "fast-join" the cluster to avoid serving an empty KV store. This fix addresses the confusing "empty ring" error response and the error log message "ring doesn't exist in KV store yet" emitted by services when there are other members present in the ring when a service starts. Those using other key<>value store options (e.g., consul, etcd) are not impacted by this bug. 
```

Reply:
>key<>value

I would call it "key-value" as in the doc.